layout/_third-party/comments/disqusjs.njk

```diff
@@ -1,6 +1,6 @@
-<link rel="stylesheet" href="{{ theme.vendors.disqusjs_css }}">
+{{ next_vendors('disqusjs_css') }}
 
 {{ next_data('disqusjs', theme.disqusjs, {
-  js: theme.vendors.disqusjs_js
+  js: theme.vendors.disqusjs_js.url
```

Comment:
I think it better to pass the whole vendor info, and edit `NexT.utils.getScript` to support `edited`, `integrity` attributes, in a position like

```javascript
  getScript: (url, {
    condition = false,
    attributes: {
      id = '',
      async = false,
      defer = false,
      crossOrigin = '',
      dataset = {},
      integrity = '', // here
      edited = false, // and here
      ...otherAttributes
    } = {},
    parentNode = null
  } = {})
```